wordpress-mobile/WordPress-Android
# Jetpack: Protect module security settings

This PR adds [Jetpack Protect module settings](https://cldup.com/awdJGSqeq6.png) capabilities. One obstacle that I ran into was that trying to update the whitelist array using a `Map<String,String>` is not possible (at least I couldn't figure out the syntax), so now Site Settings will serialize changes to a `JSONObject` parameter. Those changes are in 9642b9bd0c15e4a6d9260da647e3869ff91def1f and I toggled all settings to make sure they still mapped appropriately.

cc @oguzkocer, @maxme

To test:
1. Login to a Jetpack site
2. Go to security settings and update the whitelist (add, remove, clear it), verify changes on web
3. Toggle the `Block malicious login attempts` setting, verify changes on web
